### PR TITLE
Swap volumes for mounts

### PR DESCRIPTION
**Pull request by DavidFair**
This gives us a number of benefits
- Makes backing up the image trivial compared to docker vols
- Devs can easily delete / modify /inspect webdata
- docker-compose up "just works"

To test:
- Run `bin/boot.sh`
- Run `bin/shutdown.sh`
- Try running `docker-compose up` after bin/boot.sh (this will create nginx_net which is required)

### CODE REVIEW

**Review by martyngigg — COMMENTED** [review]
Thanks for these changes. I agree that the volume mounts are far easier to deal with, it's also how Discourse work with their shared data too.

I assume we still need the boot script so it creates the network for nginx?

**Review by martyngigg — CHANGES_REQUESTED** [review]
I'm not sure why but if I try `docker-compose up` then I get an error in the `web` container:
```
web_1                 |   File "/usr/local/lib/python3.7/importlib/__init__.py", line 127, in import_module
web_1                 |     return _bootstrap._gcd_import(name[level:], package, level)
web_1                 |   File "<frozen importlib._bootstrap>", line 1006, in _gcd_import
web_1                 |   File "<frozen importlib._bootstrap>", line 983, in _find_and_load
web_1                 |   File "<frozen importlib._bootstrap>", line 965, in _find_and_load_unlocked
web_1                 | ModuleNotFoundError: No module named 'wsgi'
```
I've actually never understood why the web container exists. It just holds the built code but that could be in an image.

**Comment by DavidFair** [comment]
This was because we're mounting an empty dir over the working dir

Me and Daniel have fixed this, we also bumped Django to 3.2 (the next LTS) as the current one is EOL this quarter and has already broken with newer `psycopg2-binary`

**Review by DanielMurphy22 — APPROVED** [review]
Works really well! It's great the css works again.

Not related to this PR but we don't seem to look at the users name on error reports on the django server. No real problem because it possible to check on adminer!
Screenshot 2022-01-26 at 09 23 15

**Comment by DavidFair** [comment]
> Works really well! It's great the css works again.
> 
> Not related to this PR but we don't seem to look at the users name on error reports on the django server. No real problem because it possible to check on adminer! Screenshot 2022-01-26 at 09 23 15

Can you open an issue?
All we need to do is add that field to the list of displayed fields in Django when we next look at it

**Comment by martyngigg** [comment]
> > With a straight `docker-compose` up I'm getting:
> > `` ERROR: Network nginx_net declared as external, but could not be found. Please create the network manually using `docker network create nginx_net` and try again. ``
> 
> This is expected, the host network is called `nginx_net`. The boot script now simply creates that external network for us mirroring our config on production

Thanks. I understand now, the straight `docker-compose up` will now just work assuming the `nginx_net` network is created so they can automatically be brought up?

File: docker-compose.yml [hunk 1]
```diff
@@ -1,20 +1,18 @@
-version: '3.2'
+version: "3.2"
 services:
   postgres:
     image: postgres:latest
```
Comment (martyngigg) [inline docker-compose.yml:4]: This is more of a note/question and I won't hold this up as it hasn't changed. I thought I would try to setup a local copy of the db so I rsync'd the `pgdata` directory from linode to my machine. When I try to start the server the postgres container won't start as it says there is a mismatch between the version of postgres running and the data that is in the directory.

Should we be pinning the major version and not using latest for reproducibility?

**Review by martyngigg — APPROVED** [review]
Confirmed this works for me too.

I've got one question but it's not about the changes here.

**Pull request merged**